modules/geochemistry/include/utils/GeochemistryUnitConverter.h

```diff
@@ -0,0 +1,101 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+#include "MooseTypes.h"
+#include "PertinentGeochemicalSystem.h"
+
+/**
+ * Utilities to convert to and from mole units
+ */
+namespace GeochemistryUnitConverter
+{
+enum class GeochemistryUnits
```

Comment:
```suggestion
enum class GeochemistryUnit
```

We usually keep these singular - do so if you desire consistency :)

Reply:
Sure, done.